tutorials/1ch.md

```diff
@@ -48,22 +48,22 @@ types that Cloud Haskell needs at a minimum in order to run.
 
 In `app/Main.hs`, we start with our imports:
 
-{% highlight haskell %}
+```haskell
 import Network.Transport.TCP (createTransport, defaultTCPParameters)
 import Control.Distributed.Process
 import Control.Distributed.Process.Node
-{% endhighlight %}
+```
```

Comment:
Is there a reason the code highlighting has all these lines?

<img width="1502" height="564" alt="Image" src="https://github.com/user-attachments/assets/a7794901-20c9-4315-9446-00f2bced1224" />